content/docs/demos/prometheus_grafana.md

```diff
@@ -1,49 +1,61 @@
 ---
-title: "Prometheus & Grafana"
-description: "Describes basic steps to get up and running a Prometheus and Grafana stack with OSM specific configuration and Dashboards"
+title: "Integrate OSM with Prometheus and Grafana"
```

Comment:
The other OSM demos have a slightly different format. For example, all demos begin like this:
![image](https://user-images.githubusercontent.com/64559656/143059456-a77b96e8-a81f-4aff-9138-9c0b6faa7432.png)

And the `Integrate OSM with Prometheus and Grafana` demo is formatted like this:
![image](https://user-images.githubusercontent.com/64559656/143060863-99e108db-868a-4392-adc9-1a07dc8b4f3c.png)

I am not sure what the best formatting option is and this doesn't need to be addressed in this PR, but I think a consistent layout is helpful.

Reply:
agreed. let me know what you think of the formatting for this demo and i can update the others as well

Reply:
I like including the title and using the subtitle font for the description as long as it is brief.